unit-tests/py/rspy/devices.py

```diff
@@ -730,7 +730,7 @@ def _get_port_by_loc( usb_location ):
                     acroname.connect()
                     if platform.system() == 'Linux':
                         _acroname_hubs = set( acroname.find_all_hubs() )
-                except NoneFoundError as e:
+                except acroname.NoneFoundError as e:
```

Comment:
What does this change?

Reply:
It fixes an exception -- `NoneFoundError` is defined inside `acroname` so it wasn't found before.